src/goose/utils.clj

```diff
@@ -40,12 +40,12 @@
   "Sleep for given seconds, multiplied by count.
   Sleep duration: (seconds * count) + jitters"
   ([sec]
-   (Thread/sleep (sec->ms sec)))
+   (Thread/sleep ^long (sec->ms sec)))
```

Comment:
We've missed adding a type-hint in multiple places inside our code. Should we add a type-hint everywhere?

Reply:
resolved